server.py

```diff
@@ -0,0 +1,23 @@
+from flask import Flask, jsonify
+from flask import send_from_directory
+
+app = Flask(__name__)
+
+
+@app.route("/version", methods=['GET'])
+def database():
+    try:
+        return send_from_directory(filename="database")
+    except Exception, e:
+        return(str(e))
+
+@app.route("/<charts>", methods=['GET'])
+def download(charts):
+
+    try:
+        return send_from_directory(directory=charts, filename=charts+'.jpeg')
```

Comment:
@Walker60 you should probably also commit this `charts` directory if you have it so this script would not crash if this repo was pulled as is. Or is that directory to be filled in with our processed charts?